bin/MessagesBundle_zh.properties

```diff
@@ -216,6 +216,22 @@ Copy_split = Copy, split non standard macros
 Save_split = Save, split non standard macros
 Font_size_invalid=The font size of the text associated to graphic elements should be positive. 
 lib_in_dir=Libraries in directory
+
+Theme_management=Theme and color management
```

Comment:
Put all the untranslated resources at the end of the file. They are easier to find for those who would like to work on them.